src/depth-to-rgb-calibration.cpp

```diff
@@ -120,8 +138,22 @@ rs2_calibration_status depth_to_rgb_calibration::optimize(
             AC_LOG( DEBUG, DISABLE_RS2_CALIBRATION_CHECKS << " is on; continuing" );
         }
 
+        if (!should_continue())
+        {
+            call_back(RS2_CALIBRATION_FAILED);
```

Comment:
This will cause the FAILED callback to happen twice, no?

Reply:
Yes, I removed the call_back call